libnavigation-util/src/main/java/com/mapbox/navigation/utils/internal/maneuver/ManeuverIconHelper.kt

```diff
@@ -460,4 +462,18 @@ object ManeuverIconHelper {
             roundaboutAngle > TOP_ROUNDABOUT_ANGLE_LIMIT -> TOP_ROUNDABOUT_ANGLE_LIMIT
             else -> roundaboutAngle
         }
+
+    /**
+     * Provide a generic roundabout resource.
+     * @param drivingSide can be [ManeuverModifier.RIGHT] or [ManeuverModifier.LEFT]
+     */
+    @JvmStatic
+    @DrawableRes
+    fun provideGenericRoundabout(drivingSide: String): Int {
+        return if (drivingSide == ManeuverModifier.RIGHT) {
+            R.drawable.mapbox_roundabout_generic_left
+        } else {
+            R.drawable.mapbox_roundabout_generic_right
+        }
```

Comment:
```suggestion
        return if (drivingSide == ManeuverModifier.RIGHT) {
            R.drawable.mapbox_roundabout_generic_right
        } else {
            R.drawable.mapbox_roundabout_generic_left
        }
```

Reply:
Blocking because of this mixup.

Reply:
resources are right. The name shows directions of roundabout's arrows. When you drive right side roads roundabout arrows will be left rounded(counterclock-wise). Opposite for left-side driving. 

as reference https://en.wikipedia.org/wiki/Comparison_of_European_road_signs 

To avoid misunderstanding we can rename resources as `mapbox_roundabout_generic_driving_side_right` and `mapbox_roundabout_generic_driving_side_left`

Reply:
Ah, got it! In this case, yes, let's rename them :+1: